errtracker/errtracker.go

```diff
@@ -160,6 +168,104 @@ func detectVirt() string {
 	return strings.TrimSpace(string(output))
 }
 
+func journalError() string {
+	// TODO: look into using systemd package (needs refactor)
+	output, err := exec.Command("journalctl", "--utc", "--boot", "--priority=warning..err", "--lines=1000").CombinedOutput()
```

Comment:
Did you confirm that this works on 14.04?

Do we have any privacy policy that states this is collected?

Reply:
Will `-b 0` (current boot) work on 14.04? Also, you could override `TZ=UTC` through env.